src/App.tsx

```diff
@@ -111,8 +113,8 @@ function App() {
                             <Route path='/peers' exact component={Peers}/>
                             <Route path="/add-peer" component={AddPeer}/>
                             <Route path="/setup-keys" component={SetupKeys}/>
-                            {/*<Route path="/acls" component={AccessControl}/>
-                            <Route path="/activity" component={Activity}/>*/}
+                            <Route path="/acls" component={AccessControl}/>
```

Comment:
Codacy has a fix for the issue: [Replace `····························<Route·path="/acls"·component={AccessControl}` with `··············<Route·path="/acls"·component={AccessControl}·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
              <Route path="/acls" component={AccessControl} />
```